app/views/contests/_admin.html.erb

```diff
@@ -16,3 +16,11 @@
 <% end %>
 </ul>
 
+<% if policy(Contest).export? %>
+  <br>
+  <b> Export Contest</b>
+  <%= form_tag( export_contest_path(@contest), :method => :get) do %>
+    <%= submit_tag "Download" %>
+  <% end %>
+<% end %>
+
```

Comment:
I'd prefer for the export UI and the existing admin UI to move into "info" tab (https://github.com/NZOI/nztrain/pull/123#discussion_r578550681), I'll make a separate PR.